src/app/view/components/login/dot-login-component/dot-login.component.scss

```diff
@@ -0,0 +1,91 @@
+@import "variables";
+@import "mixins";
+@import "../styles/primeng-theme/utils/theme-variables";
+
+/* Login box */
+#login-container {
+    @include box_shadow(5);
+    background-color: $white;
+    border-radius: $border-radius * 3;
+    padding: $basic-padding * 10 $basic-padding * 10 $basic-padding * 8;
+    width: 360px;
+    box-sizing: content-box;
+    margin: 0 auto;
+
+    @media (max-width: $screen-sm-min) {
+        padding: $basic-padding * 5 $basic-padding * 5 $basic-padding * 3;
+    }
+}
+
+.login__header {
+    display: flex;
+}
+
+.login__logo {
+    flex-grow: 1;
+    align-self: center;
+    display: flex;
+
+    img {
+        width: 102px;
+        height: 19px;
+    }
+}
+
+.login__title {
+    color: $black;
+    font-size: rem-cal(34px);
+    margin: $basic-padding * 9 0 $basic-padding * 6 0;
+}
+
+.login__password-settings {
+    display: flex;
+
+    p-checkbox {
+        flex-grow: 1;
+    }
+
+    a {
+        align-self: center;
+    }
+}
+
+.login__footer {
+    color: $gray-light;
+    text-align: center;
+    text-transform: uppercase;
+    font-size: $font-size-x-small;
+}
+
+.login {
+    display: flex;
+    flex-direction: column;
+    height: 100%;
+    justify-content: center;
+
+
+    input.ui-inputtext {
+        display: block;
+        width: 100%;
+    }
+
+    md-input {
+        overflow: inherit;
+        width: 100%;
+    }
+}
+
+.login__container {
+    @extend #login-container;
+}
+
+.login__button {
+    margin-right: $basic-padding;
+}
+
+a[actionlink] {
```

Comment:
this should not go here, but where ?

Reply:
No. This should be global.

Reply:
No, we need to make this global, not sure where tho, we have to look the files.